docs/admin/config.rst

```diff
@@ -831,6 +831,33 @@ List for credentials for Bitbucket servers.
    :ref:`vcs-bitbucket-server`,
    `Bitbucket: HTTP access token <https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html>`_
 
+.. setting:: DEVOPS_CREDENTIALS
+
+DEVOPS_CREDENTIALS
+------------------
+
+.. versionadded:: 5.2
+
+List for credentials for Azure DevOps servers.
+
+.. code-block:: python
+
+    DEVOPS_CREDENTIALS = {
+        "dev.azure.com": {
+            "username": "project-name",
+            "token": "your-api-token",
+            "organization": "organization-name",
+        },
+    }
+
+.. include:: /snippets/vcs-credentials-devops.rst
```

Comment:
The sole purpose of the snippet is to be able to reuse it. If you can't use the generic description here, just include it here directly.

Reply:
I've rewritten this to better work with different services, see https://en.wikipedia.org/wiki/DevOps. That should allow you to reference existing docs while documenting service specifics properly.

Reply:
Thanks for taking a look, I knew I was missing something. I've inlined this in the appropriate place and tried to rewrite the section so it reads better and describes the options adequately :)